idp/keycloak/keycloak.go

```diff
@@ -0,0 +1,83 @@
+// Copyright 2017 Canonical Ltd.
```

Comment:
s/2017/2020/ ?

Reply:
Original code was the google idp which was Copyright 2017 Canonical. 

I do have a signed contributor agreement, but this is my first contribution to Canonical, so I'm not sure I did this right?